NDB.Covid19/NDB.Covid19/ExposureNotifications/ExposureNotificationHandler.cs

```diff
@@ -175,10 +175,19 @@ public Task<DailySummaryConfiguration> GetDailySummaryConfigurationAsync()
             return Task.FromResult(dsc);
         }
 
-        public Task ExposureStateUpdatedAsync(IEnumerable<ExposureWindow> windows, IEnumerable<DailySummary> summaries)
+        public async Task ExposureStateUpdatedAsync(IEnumerable<ExposureWindow> windows, IEnumerable<DailySummary> summaries)
         {
             Debug.WriteLine("ExposureStateUpdatedAsync is called");
-            throw new NotImplementedException();
+            bool shouldSendMessage = false;
+            foreach (DailySummary dailySummary in summaries)
+            {
+                if (ExposureDetectedHelper.RiskInDailySummaryAboveThreshold(dailySummary))
+                {
+                    shouldSendMessage = true;
+                    break;
+                }
+            }
```

Comment:
That's fine for now I guess, but I think we can do even more advanced calculation/scoring in the end :) We will see :)

Reply:
@jonasofrich 
Infectiousness.None and ReportType.Unknown and ReportType.Revoked should be removed from configuration as EN throws exceptions if we use them

Reply:
@nc-kano I am making a PR with that